packages/orbit-components/package.json

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
Nothing new in this `package.json` IIRC, I just extracted the parts of the root `package.json` that were related to `orbit-components`.